### PR TITLE
Dbms mongo 3

### PR DESCRIPTION
**Pull request by Vaibhavpratapsingh22**
Chapter - 3 created 🏎️

### CODE REVIEW

**Review by udeysingh95 — CHANGES_REQUESTED** [review]
Overall great work. 
There is some issue with the use of "quote" style formatting where "code" style formatting is expected. After that improvement - it looks good to go from my side. 
I would appreciate your review here @giteshk11. 
and @Dhillon-in your review for the completeness and the understandability of code - as you were looking forward to learning it. 
You can read and answer a simple question - was it helpful for you to learn about MongoDB collections or not?

File: 5-dbms/9-mongodb/3-collections.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+**Note**: Drop() method will return true, if the selected collection is dropped successfully, otherwise it will return false.
+
+
+
+## Practical 
+Now from above we have learned and understood the operations of database and collections. Now it's time to implement them on your system. As by implementing these commands the concept will get more clear and will be saved in the memory for a long period of time.
+
+### 1. Database Commands
+
+<img src="https://user-images.githubusercontent.com/54719422/91654231-c3f00c80-eac4-11ea-9c48-582d2fcee2e2.png " height="" width="">
```
[the remark this answers is not in the document]
Comment (SaarthakDeshmukh) [reply 5-dbms/9-mongodb/3-collections.md:145]: For me, it's loading properly.

[the remark this answers is not in the document]
Comment (Vaibhavpratapsingh22) [reply 5-dbms/9-mongodb/3-collections.md:145]: I will upload it once more.

**Review by udeysingh95 — APPROVED** [review]

**Comment by udeysingh95** [comment]
Let's wait for other reviews here.

**Review by SaarthakDeshmukh — CHANGES_REQUESTED** [review]
Overall the readability of the PR was not at an issue. Great Work there! just a few structural changes.

File: 5-dbms/9-mongodb/3-collections.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+So, now we have understood that we have to make a database before making a collection. 
+
+## Database
+We can perform two operations on database as follow:
+
+### 1. Create Database
+In MongoDB we use **use DATABASE_NAME** is used to create database. The command will create a new database if it doesn't exist, otherwise it will return the existing database.
+
+**Syntax**:\
```
Comment (SaarthakDeshmukh) [inline 5-dbms/9-mongodb/3-collections.md:13]: The description is not in line and below it is in the line. Try to follow one structure.

File: 5-dbms/9-mongodb/3-collections.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+    db.createCollection(name, options)
+
+In the command, name is name of collection to be created. Options is a document and is used to specify configuration of collection.
+
+<img src="https://user-images.githubusercontent.com/54719422/91653711-b6388800-eac0-11ea-9335-59b8dc7137fa.png" height="" width="">
+
+Options parameter is optional, so you need to specify only the name of the collection. Following is the list of options you can use as shown:
+
+<img src="https://user-images.githubusercontent.com/54719422/91653771-28a96800-eac1-11ea-9056-efe10f48190a.png" height="" width="">
```
Comment (SaarthakDeshmukh) [inline 5-dbms/9-mongodb/3-collections.md:88]: Images can be placed at the center. It would look nice.

File: 5-dbms/9-mongodb/3-collections.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+    db
+    shopdb
+
+**If you want to check your databases list**: use the command show dbs.
+
+    show dbs
+    local     0.78125GB
+    test      0.23012GB
+
+**Note**:Your created database (shopdb) is not present in list. To display database, you need to insert at least one document into it.
```
Comment (SaarthakDeshmukh) [inline 5-dbms/9-mongodb/3-collections.md:34]:
```suggestion
**Note**: Your created database (shopdb) is not present in the list. To display a database, you need to insert at least one document into it.
```

**Review by SaarthakDeshmukh — APPROVED** [review]

**Pull request merged**